docs/configuration/downstream/pull_from_upstream.md

```diff
@@ -54,6 +57,14 @@ This will take the Packit configuration file from the default branch of the dist
 
     /packit pull-from-upstream --with-pr-config
 
+
+`pull-from-upstream` triggered by the Upstream Release Monitoring automatically handles the Bugzilla created by Upstream
+Release Monitoring (by default adds `Resolves` to changelog/commit and exposes `PACKIT_RESOLVED_BUGS` to the `changelog-entry` and `commit-message`
+actions). In case of retriggering, you need to specify the bugs like this:
```

Comment:
Do you think we should/can suggest a way to find the Bugzilla?

Reply:
good point!

Reply:
Thanks!